packages/wasm/crate/src/keys.rs

```diff
@@ -181,3 +181,24 @@ pub fn forwarding_addr_inner(sequence: u16, account: Option<u32>, fvk: &FullView
 
     address
 }
+
+#[wasm_bindgen(getter_with_clone)]
+pub struct TransparentAddrResponse {
+    /// The raw (binary) transparent address
+    pub address: Vec<u8>,
+    /// The t-address encoding of the transparent address
+    pub encoding: String,
+}
+
+/// Returns the "truncated" address (t-addr) associated with the account.
+#[wasm_bindgen]
+pub fn get_transparent_address(full_viewing_key: &[u8]) -> WasmResult<TransparentAddrResponse> {
+    let fvk: FullViewingKey = FullViewingKey::decode(full_viewing_key)?;
+    let encoding = fvk.incoming().transparent_address();
+    let address: Address = encoding.parse().expect("encoding transparent address");
```

Comment:
maybe `Address::from_str(&encoding).expect("encoding transparent address")` is better here?